### PR TITLE
change the l10n the schedule strategy

### PR DESCRIPTION
**Pull request by tigershi**
the tester requested that send to GRM source immediately after source merge to local bundle. there configure scanning the GRM source directory every three second

### CODE REVIEW

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncGrmSourceServiceImpl.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public synchronized void sendSourceToGRM() {
      * scan the GRM cached source file DIR and process the cached source file
      * @throws IOException 
      */
-	private synchronized void processGRMQueueFiles() throws IOException {
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncGrmSourceServiceImpl.java:86]: why remove synchronized?

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncI18nSourceServiceImpl.java [hunk 2]
```diff
@@ -67,7 +67,7 @@
 	 *
 	 */
 	@Override
-	public synchronized void sendSourceToI18n() {
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncI18nSourceServiceImpl.java:70]: why remove synchronized?

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncLocalBundleServiceImpl.java [hunk 3]
```diff
@@ -78,7 +78,7 @@ private void processSendFilePath(String basePath, File source) throws IOExceptio
 	 * Synchronize the updated source to local bundle file
 	 */
 	@Override
-	public void mergeSourceToLocalBundle() {
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncLocalBundleServiceImpl.java:81]: why add synchronized?

Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncLocalBundleServiceImpl.java:81]: There are no thread safety problem

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncI18nSourceServiceImpl.java (hunk 2 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncI18nSourceServiceImpl.java:70]: There are no thread safety problem

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncGrmSourceServiceImpl.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SyncGrmSourceServiceImpl.java:86]: There are no thread safety problem

**Review by linr211 — APPROVED** [review]

**Review by Xiaochao8 — APPROVED** [review]

**Pull request merged**